docarray/array/mixins/io/from_gen.py

```diff
@@ -129,6 +134,17 @@ def from_csv(cls: Type['T'], *args, **kwargs) -> 'T':
         """
         return cls._from_generator('from_csv', *args, **kwargs)
 
+    def __from_csv(cls: Type['T'], *args, **kwargs) -> 'T':
+        """
+        # noqa: DAR101
+        # noqa: DAR102
+        # noqa: DAR201
+        """
+        warnings.warn(
```

Comment:
I am not sure if we should raise a `NotExistingMethod` exception or something like that @samsja , something that is the same as what `pandas` do.

Reply:
Yes I agree as well that we should raise an error rather than a warning